arctic/arctic.py

```diff
@@ -202,6 +203,42 @@ def list_libraries(self):
                         libs.append(coll[:-1 * len(self.METADATA_COLL) - 1])
         return libs
 
+    @mongo_retry
+    def invalidate_libraries_cache(self):
+        """
+        Deletes the data in the collection used to cache the result
+        of list_libraries
+        :return:
+        """
+        self._conn.cache_db.cached_lib_names.drop()
+
+    @mongo_retry
+    def list_libraries_cached(self):
+        """
+        Returns
+        -------
+        List of Arctic library names from a cached collection (global per mongo cluster) in mongo.
+        Long term list_libraries should have a use_cached argument.
+        """
+        mongo_db = self.__conn.cache_db
+        col_name = 'cached_lib_names'
```

Comment:
Even if we do have that we can have a separate database called (say) meta_db and have a new collection for each type of metadata/accounting stuff rather than reusing this.